apps/remix-ide/src/blockchain/blockchain.tsx

```diff
@@ -748,19 +748,34 @@ export class Blockchain extends Plugin {
     )
 
     web3Runner.event.register('transactionBroadcasted', (txhash) => {
-      this.executionContext.detectNetwork((error, network) => {
+      this.executionContext.detectNetwork(async (error, network) => {
         if (error || !network) return
         if (network.name === 'VM') return
         const viewEtherScanLink = etherScanLink(network.name, txhash)
-
+        const viewBlockScoutLink = await getBlockScoutUrl(network.id, txhash)
         if (viewEtherScanLink) {
-          this.call(
-            'terminal',
-            'logHtml',
-            <a href={etherScanLink(network.name, txhash)} target="_blank">
-              view on etherscan
-            </a>
-          )
+          if (viewBlockScoutLink) {
+            this.call(
+              'terminal',
+              'logHtml',
+              <span className="flex flex-row">
+                <a href={etherScanLink(network.name, txhash)} className="mr-3" target="_blank">
+                  view on Etherscan
+                </a>
+                <a href={viewBlockScoutLink} target="_blank">
+                  view on Blockscout
+                </a>
+              </span>
+            )
+          } else {
```

Comment:
Etheracan should stay anyway

Reply:
I revised the code so in the event there isn't an etherscan link, there could be a blockscout link. There both are not available, then nothing shows.